commcare_connect/users/models.py

```diff
@@ -96,16 +96,18 @@ class CredentialType(models.TextChoices):
     class Meta:
         unique_together = ("user", "opportunity", "credential_type")
 
-    @property
-    def delivery_level_num(self):
-        if self.credential_type == self.CredentialType.LEARN:
+    @classmethod
```

Comment:
Given the implementation it made more sense to make these two methods class methods.